doc/releasenotes/14_0_0_summary.md

```diff
@@ -83,17 +99,20 @@ $ vtctl -- --topo_implementation etcd2 AddCellInfo --root "/vitess/global"
 
 The default value for `--heartbeat_on_demand_duration` is zero, which means the flag is not set and there is no change in behavior.
 
-When `--heartbeat_on_demand_duration` has a positive value, then heartbeats are only injected on demand, per internal requests. For example, when `--heartbeat_on_demand_duration=5s`, the tablet starts without injecting heartbeats. An internal module, like the lag throttle, may request the heartbeat writer for heartbeats. Starting at that point in time, and for the duration (a lease) of `5s` in our example, the tablet will write heartbeats. If no other requests come in during that duration, then the tablet then ceases to write heartbeats. If more requests for heartbeats come while heartbeats are being written, then the tablet extends the lease for the next `5s` following up each request. Thus, it stops writing heartbeats `5s` after the last request is received.
+When `--heartbeat_on_demand_duration` has a positive value, then heartbeats are only injected on demand, based on internal requests. For example, when `--heartbeat_on_demand_duration=5s`, the tablet starts without injecting heartbeats.
+An internal module, like the lag throttler, may request the heartbeat writer for heartbeats. Starting at that point in time, and for the duration (a lease) of `5s` in our example, the tablet will write heartbeats.
+If no other requests come in during that time, the tablet then ceases to write heartbeats. If more requests for heartbeats come in, the tablet extends the lease for the next `5s` following each request.
+It stops writing heartbeats `5s` after the last request is received.
 
 The heartbeats are generated according to `--heartbeat_interval`.
 
 #### Deprecation of --online_ddl_check_interval
 
 The flag `--online_ddl_check_interval` is deprecated and will be removed in `v15`. It has been unused in `v13`.
 
-#### Deprecation of --planner-version for vtexplain
+#### Removal of --gateway_implementation
 
-The flag `--planner-version` is deprecated and will be removed in `v15`. Instead, please use `--planer_version`.
```

Comment:
@systay this seems correct to me. We still have `planner-version`, we introduced and removed `planner_version`.